Makefile

```diff
@@ -34,7 +35,6 @@ deb: build
 	-t deb \
 	-v $(VERSION) \
 	--deb-no-default-config-files \
-	--deb-use-file-permissions \
```

Comment:
just to update on this, until getting an answer in the issue (hopefully!), after a bit more testing, I think `--deb-use-file-permissions` sets uid and gid of the files in the package to the same uid and gid of the user building it. 

this explains why files are owned by `ubuntu:ubuntu` on the IDE and `kareem:kareem` on my Ubuntu (it sets uid to 1000 and gid to 1000). to even confirm more, `sudo make deb` sets the ownership to `root:root`.